docs/learn/bossroom/getting-started-boss-room.md

```diff
@@ -6,200 +6,275 @@ description: Learn more about installing and running the Boss Room game sample.
+**Note**: If you’re using Mac to run multiple instances of the same app, you need to use the command line. Run `open -n BossRoom.app`.
 :::
 
-The Authentication API doesn't distinguish multiple instances of the same project running on the same machine and logs in the same user in all those different instances. Both ParrelSync clones and actual game builds are affected and can mess up testing the game locally.
+#### Multiplayer over Internet
 
-However, Authentication supports [Profiles](https://docs.unity.com/authentication/ProfileManagement.html) that allows different users existing on the same physical machine. To test locally, we need both builds and editor players to be able to switch to different Profiles.
+Use the following instructions to test Boss Room with multiple players over the Internet.
 
-The [`ProfileManager`](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/main/Assets/Scripts/Utils/ProfileManager.cs) class wraps the logic for deciding what Profile you should be using, if any. In builds, the command-line argument `-AuthProfile` specifies the new profile id. When iterating in the editor, ParrelSync can use its `CloneManager` custom arguments to decide what user profile to use. Users can also change profile using the in-game menu.
+First, build an executable and distribute it to all players.
 
-`ProfileManager.Profile` generates the custom `InitializationOptions`:
+:::tip
+**Tip**: It is possible to connect between multiple instances of the same executable OR between executables and the Unity Editor that you used to create the executable.
+:::
 
-```
-var unityAuthenticationInitOptions = new InitializationOptions();
-var profile = ProfileManager.Profile;
+Next, you need to set up a relay. Running the Boss Room over the internet currently requires setting up a relay:
+
+* Boss Room provides an integration with [Unity Relay](../../relay/relay.md). You can find our Unity Relay setup guide [here](../../relay/relay.md).
+* Alternatively you can use Port Forwarding. The [https://portforward.com/](https://portforward.com/) site has guides on how to enable port forwarding on a huge number of routers.
+* Boss Room uses `UDP` and needs a `9998` external port to be open.
+* Make sure your host's address listens on 0.0.0.0 (127.0.0.1 is for local development only).
+
+### Index of resources
+
+The Boss Room sample includes resources for gameplay, game flow, connectivity, Unity services, and other tools and utilities.
+
+#### Gameplay
 
+Boss Room includes the following gameplay resources:
 
-if (profile.Length > 0)
-{
-    unityAuthenticationInitOptions.SetProfile(profile);
-}
+* Action anticipation - AnticipateActionClient() in [Assets/Scripts/Gameplay/Action/Action.cs](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/main/Assets/Scripts/Gameplay/Action/Action.cs)
```

Comment:
Not sure I understand, this PR doesn't simplify and does the reverse, it copies even more things from the readme.
On another note, do these links work in the docs page? Cause those are relative links in boss room

Reply:
It does remove some content, sure. But it does add quite a lot as well

Reply:
The goal is to remove as much duplication of information as possible and have the docs live in the place where it makes the most sense. Not to duplicate the readme on the docs website

Reply:
I must have misunderstood our converstaion earlier. I thought you suggested using the information from the README (albeit as a temporary measure) to simplify and update the Get started guide. De-duplicating the information across the site is going to be a larger effort.